rosdep/python.yaml

```diff
@@ -3096,9 +3096,10 @@ python-pylint:
   ubuntu: [pylint]
 python-pylint3:
   debian: [pylint3]
-  fedora: [python3-pylint]
```

Comment:
Fedora does provide this package https://packages.fedoraproject.org/pkgs/pylint/python3-pylint/ - I was able to just `docker run -it fedora` and then `yum install python3-pylint` with no issues - what fedora version did you test this on to rule it out?

Reply:
Thanks for checking. No, as I mentioned in OP, I had not tested on Fedora.

Update pushed.